examples/rpc/partner-plugin/handlers/ping/v1/ping_handler.go

```diff
@@ -0,0 +1,36 @@
+package handlers_mock
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/chain4travel/camino-messenger-bot/internal/metadata"
+
+	pingv1 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/ping/v1"
+)
+
+type PingResponseHandler interface {
+	Ping(ctx context.Context, request *pingv1.PingRequest) (*pingv1.PingResponse, error)
+}
+
+func NewPingResponseHandler() PingResponseHandler {
+	return &pingResponseHandler{}
+}
+
+type pingResponseHandler struct{}
+
+func (p *pingResponseHandler) Ping(ctx context.Context, request *pingv1.PingRequest) (*pingv1.PingResponse, error) {
```

Comment:
In case Ping should not differ from other examples:
```suggestion
package handlers

import (
	"context"
	"fmt"
	"log"

	"buf.build/gen/go/chain4travel/camino-messenger-protocol/grpc/go/cmp/services/ping/v1/pingv1grpc"
	pingv1 "buf.build/gen/go/chain4travel/camino-messenger-protocol/protocolbuffers/go/cmp/services/ping/v1"
	"github.com/chain4travel/camino-messenger-bot/internal/metadata"
)

var _ pingv1grpc.PingServiceServer = (*PingServiceV1Server)(nil)

type PingServiceV1Server struct{}

func (*PingServiceV1Server) Ping(ctx context.Context, request *pingv1.PingRequest) (*pingv1.PingResponse, error) {

```